---
created: '2024-11-08T19:20:50.291917+00:00'
creator: sentry
source: tests/sentry/grouping/test_grouphash_metadata.py
---
hash_basis: stacktrace
hashing_metadata: {
  "num_stacktraces": 1,
  "stacktrace_location": "exception",
  "stacktrace_type": "system"
}
---
contributing variants:
  system*
    hash: "90888e813b09fa25061af2883c0fb9bd"
    component:
      system*
        exception*
          stacktrace*
            frame*
              module*
                "django.core.handlers.base"
              function*
                "get_response"
              context-line*
                "response = wrapped_callback(request, *callback_args, **callback_kwargs)"
            frame*
              module*
                "django.views.generic.base"
              function*
                "view"
              context-line*
                "return self.dispatch(request, *args, **kwargs)"
            frame*
              module*
                "django.utils.decorators"
              function*
                "_wrapper"
              context-line*
                "return bound_func(*args, **kwargs)"
          type*
            "MultiValueDictKeyError"
